run.sh

```diff
@@ -1,13 +1,42 @@
 #!/bin/bash
 export MIN_JS=false
 
-ADD=""
+PROFILE="dev"
+CREATE_DB=""
+YARN="-Dskip.yarn"
 
-if [ "$1" = "--create-db" ]; then
-   ADD="-Dhbm.dev.auto=create"
+for arg in "$@"
+do
+    if [ "$arg" = "--create-db" ]; then
+       CREATE_DB="-Dhbm.dev.auto=create"
+    fi
+
+    if [ "$arg" = "--yarn" ]; then
+       YARN=""
+    fi
+
+    if [ "$arg" = "--prod" ]; then
+       PROFILE="prod"
+    fi
```

Comment:
If you're running in the `prod` profile you should also be setting `irida.db.profile=prod`

Reply:
These seem backwards to be.  If I added `--yarn` to the command line I would expect yarn to be run (or is it me that is backwards).  As to answer @tom114 above I think by default it is fine to ignore because it is so much faster.  Just make sure that it is noted somewhere that you need to run `yarn install` and one of `yarn build` or `yarn start`